server/__tests__/index.test.js

```diff
@@ -1,5 +1,8 @@
-const request = require('supertest');
-const server = require('../src/index');
+
+import '@babel/polyfill';
```

Comment:
You could just specify useBuiltIns: 'usage' in .babelrc
![image](https://user-images.githubusercontent.com/31539366/53121705-ccd05b80-357a-11e9-9927-f391883f0c40.png)

Reply:
Good catch. Thanks!